CHANGELOG.md

```diff
@@ -9,6 +9,7 @@
 - Add enableAutoBreadcrumbTracking option (#1958)
 - Automatic nest spans with the UI life cycle (#1959)
 - Upload frame rendering timestamps to correlate to sampled backtraces (#1910)
+- Remove variable values from CoreData queries (#1982)
```

Comment:
`h`: This should move up to the unreleased section.

Reply:
Merge 😕